frontend/src/pages/GettingStarted.tsx

```diff
@@ -32,7 +32,7 @@ const PAGE_CONTENT_MD = `
 
 ## Build your own pipeline
 
-Build an end-to-end ML pipeline with TensorFlow Extended (TFX) [**Start Here!**](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Ftensorflow%252Ftfx%252Fmaster%252Fdocs%252Ftutorials%252Ftfx%252Ftemplate.ipynb) (Alpha)
+Build an end-to-end ML pipeline with TensorFlow Extended (TFX) [**Start Here!**](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Ftensorflow%252Ftfx%252Fmaster%252Fdocs%252Ftutorials%252Ftfx%252Ftemplate.ipynb) (Experimental)
 
```

Comment:
I prefer just remove the Alpha or Experimental

Reply:
Updated accordingly, I will hold off until noon, if there are no objections will push through.